src/app/AlwaysOn.Shared/TelemetryExtensions/AppInsightsRequestHandler.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.DataContracts;
+using Microsoft.Azure.Cosmos;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace AlwaysOn.Shared.TelemetryExtensions
+{
+    public class AppInsightsRequestHandler : RequestHandler
```

Comment:
can we rename that to include "CosmosDb" in the name somehow? Otherwise looking just at the name, its really not clear what kind of request handler this is

Reply:
I was thinking about that, but since `RequestHandler` is coming from Cosmos DB I decided to keep it simple. But ofc can do.

Reply:
Yeah but you need to run Intellisense etc to see where RequestHandler is coming from. So I'd opt to rename it